### DISCUSSION TITLE
Feature Request: Play sound when done

### DISCUSSION THREAD

**Discussion by barepixels** [MoonRide303/Fooocus-MRE#18] [Ideas]
One of the features in A1111 that I find useful is the ability to play a sound when done generating.  All I have to do is drop a "notification.mp3" into the root of A1111 folder.  Often, I would set a large batch to generate then go surf the web or play a game on my phone while waiting for the notification sound.  Attached is the mp3 that I use
[notification.zip](https://github.com/MoonRide303/Fooocus-MRE/files/12420519/notification.zip)

**Comment by barepixels** [comment]
It seem [runew0lf](https://github.com/runew0lf)/[RuinedFooocus](https://github.com/runew0lf/RuinedFooocus) now can Play sound when done!!!!

Here is what he wrote:
14) Displays time taken for each render in the console. If notification.mp3 exists in the root directory, this will play when the generations are complete.

@MoonRide303 can you merge his codes over

**Comment by MoonRide303** [comment]
@barepixels Done (commit [071eb99](https://github.com/MoonRide303/Fooocus-MRE/commit/071eb9908f6dc0e7ffc2b65fd746dc23affc1d3d)) - I ported implementation from SD web UI (based on gr.Audio), as it's more reliable than playsound (and I'd also prefer to not add additional dependencies unless it's absolutely necessary).
